src/apis/stockx/asks/index.js

```diff
@@ -51,10 +51,10 @@ export default class Asks extends Base {
 
     checkStatus(res);
 
-    const { body } = res;
-    const { PortfolioItem: { chainId, skuUuid }} = body;
-    
-    return { chainId, skuUuid };
+    const body = JSON.parse(res.body);
```

Comment:
Is the call to `JSON.parse()` needed? AFAIK rp takes care of the transformer for you depending on the `content-type` in the response (and a few other requirements).